plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -535,6 +536,40 @@ public void testAddColumnWithCommentSpecialCharacter(String comment)
         }
     }
 
+    @Test
+    public void testAddColumnWithDecimal()
+    {
+        testAddColumnWithDecimal(DecimalType.createDecimalType(14, 5));
+        testAddColumnWithDecimal(DecimalType.createDecimalType(35, 5));
+    }
+
+    protected void testAddColumnWithDecimal(DecimalType decimal)
+    {
+        String tableName = "test_add_column_with_decimal" + randomNameSuffix();
+
+        assertUpdate("CREATE TABLE IF NOT EXISTS " + tableName + " (" +
+                "id INT WITH (primary_key=true), " +
+                "a_varchar VARCHAR" +
+                ") WITH (" +
+                " partition_by_hash_columns = ARRAY['id'], " +
+                " partition_by_hash_buckets = 2" +
+                ")");
+
+        String displayName = decimal.getDisplayName();
+        assertUpdate("ALTER TABLE " + tableName + " ADD COLUMN b_decimal " + displayName);
+        assertThat(getDecimalType(tableName)).isEqualTo(displayName);
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    protected String getDecimalType(String tableName)
```

Comment:
Please remove this method and use `BaseConnectorTest#getColumnType` instead.

Reply:
done~